README.md

```diff
@@ -255,7 +245,7 @@ For example, to create a Docker image of the agent, tagged `v1.0.0`:
 
 and then the Docker image `v1.0.0` of the agent can be started  by
 ```shell
-docker run -v $(pwd)/config.yaml:/config.yaml  -p 55678:55678  ocagent:v1.0.0
+docker run --rm -it -v $(pwd)/ocagent-config.yaml:/conf/ocagent-config.yaml -p 55678:55678 -p 55679:55679 ocagent:v1.0.0
```

Comment:
You need to pass `--config=/conf/ocagent-config.yaml` on the command-line

Reply:
fixing